Userland/Libraries/LibWeb/DOM/Document.idl

```diff
@@ -57,6 +57,8 @@ interface Document : Node {
 
     attribute DOMString cookie;
 
+    attribute DOMString bgColor;
```

Comment:
These should have `[CEReactions, LegacyNullToEmptyString]` per the spec: https://html.spec.whatwg.org/#Document-partial

Reply:
Also while you're here, you could also implement `fgColor`, `linkColor`, `vlinkColor`, `alinkColor` in exactly the same way as `bgColor` :slightly_smiling_face: